src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-sync-primary-zone/rgw-sync-primary-zone.component.html

```diff
@@ -1,15 +1,27 @@
 <ul class="pb-5">
-  <li><i [ngClass]="[icons.large2x, icons.reweight]"
-         class="pt-2"></i></li>
+  <li>
+    <svg [cdsIcon]="icons.reweight"
+         [size]="icons.size20"
+         class="cds--btn__icon cds-size-32"></svg>
```

Comment:
```suggestion
         class="cds--btn__icon cds-icon--size"></svg>
```
`cds-size-32` this needs better naming. There is no 32 in used css.

Reply:
Same for below